### ISSUE TITLE
Unable to find services.yaml for the elastic integration

### ISSUE THREAD

**Issue by robin13** [legrego/homeassistant-elasticsearch#72]
In the logs I see this warning:

```
Unable to find services.yaml for the elastic integration
```

From the docs, this seems it is nothing critical, but rather just a missing description of the service provided by elasticsearch:
https://developers.home-assistant.io/docs/en/dev_101_services.html#service-descriptions

What would an appropriate entry in the `services.yaml` file be for the elastic integration?

**Comment by legrego** [comment]
Thanks for raising this @robin13 -- I must have overlooked this change to HASS.

I think an appropriate `services.yaml` would look something like this (untested):

```yaml
publish_events:
  # Description of the service
  description: Publishes all queued events to Elasticsearch.
```

This service call is not configurable, so there are no fields supplied.

**Comment by robin13** [comment]
Yes - this worked. :)
https://github.com/legrego/homeassistant-elasticsearch/pull/73

**Comment by legrego** [comment]
Closed via #73
